sktime/forecasting/trend/_polynomial_trend_forecaster.py

```diff
@@ -81,13 +84,22 @@ class PolynomialTrendForecaster(BaseForecaster):
         "ignores-exogeneous-X": True,
         "requires-fh-in-fit": False,
         "handles-missing-data": False,
+        "capability:pred_var": True,
+        "capability:pred_int": True,
     }
 
-    def __init__(self, regressor=None, degree=1, with_intercept=True):
+    def __init__(
+        self, regressor=None, degree=1, with_intercept=True, prediction_intervals=False
+    ):
         self.regressor = regressor
         self.degree = degree
         self.with_intercept = with_intercept
         self.regressor_ = self.regressor
+        self.prediction_intervals = prediction_intervals
+        # prediction_intervals : bool, default=False
+        # By default, the extra information needed to later generate the prediction
+        # intervals is not calculated. If set to True, the extra information is
+        # calculated and stored in the forecaster.
         super().__init__()
```

Comment:
The failures are since the tag is inconsistent in the case `prediction_intervals=False`, so the tag needs to be set dynamically in the `__init__`.

I've added the necessary line to save time.